include/maidsafe/crux/detail/sequence_number.hpp

```diff
@@ -36,6 +36,7 @@ class sequence_number {
 public:
     sequence_number();
     sequence_number(const sequence_number&);
+    sequence_number(sequence_number&&);
```

Comment:
Is this necessary? There is no implementation for it.

If it is necessary, then I think that we need to add the move assignment operator as well to observe the rule of five.